plugins/flatpak/gs-flatpak.c

```diff
@@ -3641,7 +3641,7 @@ gs_flatpak_mount_removed (GVolumeMonitor *volume_monitor,
 	GsFlatpak *self = GS_FLATPAK (user_data);
 	GHashTableIter iter;
 	gpointer key;
-	gboolean should_reload = FALSE;
+	//gboolean should_reload = FALSE;
```

Comment:
I think all the changes to this file need to be reverted since they cause a regression: if you have the USB inserted and copy an app to it, the USB category no longer gets reloaded to reflect the changes (as it did before this patch).

I don't immediately see why leaving this in place would conflict with the rest of the patch.